components/ForceVector.vue

```diff
@@ -26,26 +32,74 @@ const props = defineProps({
         y: {type: Number, required: true},
     },
     showComponents: {type: Boolean, default: false},
+    id: {
+        type: String,
+        required: true
+    },
 })
 
+const dragCircle = (event) => {
+    const stage = event.target.getStage()
+    const pointerPosition = stage.getPointerPosition()
+    const snapTolerance = 15
```

Comment:
can you define numerical constants like this up at the top of the component, just to make it easier to call them out? Also do we want a convention for how we write those, like SNAP_TOLERANCE?

Reply:
Great idea.